cmd/cc-uploader/main.go

```diff
@@ -54,12 +61,28 @@ func main() {
 
 	initializeDropsonde(logger, uploaderConfig)
 
+	// Create signal channel to listen for shutdown signals
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, os.Interrupt, syscall.SIGTERM)
+
+	// Goroutine to log any signal received (without handling non-TERM signals)
+	go func() {
+		allSignals := make(chan os.Signal, 1)
+		signal.Notify(allSignals) // Capture all signals for logging
+		for sig := range allSignals {
+			logger.Info("received-signal", lager.Data{"signal": sig.String()})
```

Comment:
Have you checked that this function does not produce too much logging output?

Reply:
Yes it does, removed it